pina/solver/ensemble_solver/ensemble_solver_interface.py

```
@@ -0,0 +1,150 @@
+"""Module for the DeepEnsemble solver interface."""
+
+import torch
+from ..solver import MultiSolverInterface
+from ...utils import check_consistency
+
+
+class DeepEnsembleSolverInterface(MultiSolverInterface):
```

Since this is an Abstract Class I would add metaclass=ABCMeta

---

isn't it inherit from the fact that abstract method are not overloaded?

---

I am positive it is as @dario-coscia says. To be 100% sure. just try to create an instance of the class. It should return an error.